src/fennel/view.fnl

```diff
@@ -290,6 +291,8 @@ Can take an options table with these keys:
   lengths
 * :max-sparse-gap (integer, default 10) maximum gap to fill in with nils in
   sparse sequential tables.
+* :preprocess (function) if present, it's called on each value in x and the
+  result is then used instead of x for pretty printing; takes the same arguments
```

Comment:
I think this wording is a bit confusing as-is, since the proprocessor will be called only on X, and then later (potentially) on each value in X as fennelview recursively outputs the results. I think we'll want to explicitly mention the recursion in there, maybe something like:
```suggestion
* :preprocess (function) if present, called on x (and recursively on each value in x),
  and the result is used for pretty printing; takes the same arguments as fennel.view
```